package.json

```diff
@@ -45,6 +45,7 @@
     "handlebars": "4.7.7",
     "httpntlm": "1.8.1",
     "js-sha512": "0.8.0",
+    "jsonwebtoken": "8.5.1",
```

Comment:
check if we can upgrade to v9
https://github.com/auth0/node-jsonwebtoken/compare/v8.5.1...v9.0.0

Reply:
Done upgraded to v9